host/4/publish-appservice-stage.yml

```diff
@@ -156,6 +156,20 @@ steps:
       docker system prune -a -f
     displayName: tag and push node images
     continueOnError: false
+  
+  - bash: |
+      set -e
+      docker pull $SOURCE_REGISTRY/node:$(PrivateVersion)-node20-appservice
+
+      docker tag $SOURCE_REGISTRY/node:$(PrivateVersion)-node20-appservice $TARGET_REGISTRY/node:$(TargetVersion)-appservice-stage$(StageNumber)
```

Comment:
This sets the default node version to node 20 which is bad practice unless totally intended.  Could upgrade some small subset of customers randomly to this version. Remove

Reply:
Not intended, removed